apps/web/components/booking/BookingDescription.tsx

```diff
@@ -0,0 +1,113 @@
+import { SchedulingType } from "@prisma/client";
+import { BookPageProps } from "pages/[user]/book";
+import { HashLinkPageProps } from "pages/d/[link]/book";
+import { AvailabilityTeamPageProps } from "pages/team/[slug]/[type]";
+import { TeamBookingPageProps } from "pages/team/[slug]/book";
```

Comment:
i've never seen these imports, does that work fine @zomars ?

Reply:
This might break on some environment. Let's switch them to relative ones.